package.json

```diff
@@ -175,6 +175,18 @@
         "command": "qute.validation.enabled.toggle.on",
         "title": "Enable Qute Validation",
         "icon": "$(eye-closed)"
+      },
+      {
+        "command": "qute.refactor.surround.with.comments",
```

Comment:
You need to write some filter because today:

 * your command are availablefor any files (ex : xml file)
 * your command is available even if the Qute LS is not started.

I suggest that you see XML https://github.com/redhat-developer/vscode-xml/blob/7accd8a0bf5d15aa0955a281b6edc9caacd2e0c4/package.json#L815

```
{
          "command": "xml.refactor.surround.with.comments",
          "when": "editorLangId in xml.supportedLanguageIds && XMLLSReady"}
}
```

For XMLLSReady, see https://github.com/redhat-developer/vscode-xml/blob/main/src/client/xmlClient.ts#L43

For xml.supportedLanguageIds, see https://github.com/redhat-developer/vscode-xml/blob/a17945188ade7f39a3d8ee0fbc75443db5ac3200/src/extension.ts#L46

Reply:
I have made some changes for this issue but I'm not sure why the command still exists even under those conditions?

Reply:
Your when is declared in the wrong place. You need to redeclare the command palette like https://github.com/redhat-developer/vscode-quarkus/blob/cc2e27697dca8c9cfbfff5b9c7bea885f5151fa6/package.json#L340